src/util/heap/layout/vm_layout_constants.rs

```diff
@@ -63,7 +63,7 @@ pub const HEAP_START: Address =
 pub const HEAP_END: Address = chunk_align_up(unsafe { Address::from_usize(0xb000_0000) });
 #[cfg(target_pointer_width = "64")]
 pub const HEAP_END: Address =
-    chunk_align_up(unsafe { Address::from_usize(0x0000_2000_0000_0000usize) });
+    chunk_align_up(unsafe { Address::from_usize(0x0000_2200_0000_0000usize) });
```

Comment:
We declare `MAX_SPACES` to be 16:
https://github.com/mmtk/mmtk-core/blob/f71d1950beee79af2150f71608975aaebd1e2572/src/util/heap/layout/heap_parameters.rs#L4

and the `HEAP_END` does not match that. So I made the change. I can change `HEAP_END` to a calculated value from `MAX_SPACES` and `MAX_SPACE_EXTENT`. So this could be clearer.

Reply:
I see

Reply:
Done in https://github.com/mmtk/mmtk-core/pull/632/commits/5c2b1a8e7f66078c67410e0b03e132cce0eecffc.